xds/src/main/java/io/grpc/xds/EnvoyServerProtoData.java

```diff
@@ -150,9 +149,9 @@ abstract static class CidrRange {
 
     abstract int prefixLen();
 
-    static CidrRange create(String addressPrefix, int prefixLen) throws UnknownHostException {
+    static CidrRange create(InetAddress addressPrefix, int prefixLen) {
```

Comment:
Why did you change the signature? `String addressPrefix` constructor seems pretty useful, given that we get a string in a proto.